### ISSUE TITLE
Cannot load with eco 6.70.0 on Paper 1.20.6 mojmap

### ISSUE THREAD

**Issue by HaHaWTH** [Winds-Studio/DeathMessages#56]
**Environment:**
Azul zulu JDK 21.0.2
Winds-Studio/Leaf 1.20.6 (mojang mapped)

Since eco 6.70.0 is released, DeathMessages' hook of eco no longer work and causes the plugin failed to load.

**Stacktrace**

```[INFO]: [DeathMessages] Enabling DeathMessages v1.4.19-SNAPSHOT

[INFO]:

[INFO]:     ____             __  __    __  ___

[INFO]:    / __ \___  ____ _/ /_/ /_  /  |/  /__  ______________ _____ ____  _____

[INFO]:   / / / / _ \/ __ `/ __/ __ \/ /|_/ / _ \/ ___/ ___/ __ `/ __ `/ _ \/ ___/

[INFO]:  / /_/ /  __/ /_/ / /_/ / / / /  / /  __(__  |__  ) /_/ / /_/ /  __(__  )

[INFO]: /_____/\___/\__,_/\__/_/ /_/_/  /_/\___/____/____/\__,_/\__, /\___/____/

[INFO]:                                                        /____/

[INFO]: [PlaceholderAPI] Successfully registered internal expansion: deathmessages [1.4.19-
SNAPSHOT]

[INFO]: [DeathMessages] PlaceholderAPI Hook Enabled!

INFO]: [DeathMessages] WorldGuard Hook Enabled!

[INFO]: [DeathMessages] PlugMan found. Adding this plugin to its ignored plugins list due to WorldGuard hook being enabled!

[ERROR]: Error occurred while enabling DeathMessages v1.4.19-SNAPSHOT (Is it up to date?)

java.lang.NoClassDefFoundError: com/willfp/libreforge/Holder
        at DeathMessages-1.4.19-
SNAPSHOT.jar/dev.mrshawn.deathmessages.DeathMessages.initHooks(DeathMessages.java:228) ~[DeathMessages-1.4.19-SNAPSHOT.jar:?]
        at DeathMessages-1.4.19-SNAPSHOT.jar/dev.mrshawn.deathmessages.DeathMessages.onEnable(DeathMessages.java:103) ~[DeathMessages-1.4.19-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:287) ~[leaf-api-1.20.6-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[leaf-1.20.6.jar:git-Leaf-"38e9d59"]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[leaf-1.20.6.jar:git-Leaf-"38e9d59"]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:518) ~[leaf-api-1.20.6-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:660) ~[leaf-1.20.6.jar:git-Leaf-"38e9d59"]
        at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:572) ~[leaf-1.20.6.jar:git-Leaf-"38e9d59"]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:670) ~[leaf-1.20.6.jar:git-Leaf-"38e9d59"]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:429) ~[leaf-1.20.6.jar:git-Leaf-"38e9d59"]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:368) ~[leaf-1.20.6.jar:git-Leaf-"38e9d59"]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1147) ~[leaf-1.20.6.jar:git-Leaf-"38e9d59"]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315) ~[leaf-1.20.6.jar:git-Leaf-"38e9d59"]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.ClassNotFoundException: com.willfp.libreforge.Holder
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:197) ~[leaf-api-1.20.6-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:164) ~[leaf-api-1.20.6-R0.1-SNAPSHOT.jar:?]
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
        ... 14 more```

**Comment by Dreeam-qwq** [comment]
Fixed in cbdfc423888e2e4711745d752a8dd508c34dd08e

**Comment by HaHaWTH** [comment]
> Fixed in [cbdfc42](https://github.com/Winds-Studio/DeathMessages/commit/cbdfc423888e2e4711745d752a8dd508c34dd08e)

😘
